docs/understanding-tracking-design/introduction-to-tracking-design/best-practices/index.md

```diff
@@ -0,0 +1,252 @@
+However, we can already see a few reusable entities that we need to track: user, cart, product, page referrer.
+The reports can also tell us what events we'll need as interactions of these entities: add product to cart, checkout step, transaction, page view.
+
+:::
+
+## Make use of our out-of-the-box schemas
+
+Snowplow trackers and data models provide a rich set of events and entities to cover common use cases.
+These include:
+
+* user and session identification,
+* device and browser information,
+* e-commerce events and entities,
+* media playback events and entities,
+* error and performance tracking,
+* and much more.
+
+It is recommended to use and build on top of these data structures in order to save effort and make use of the existing Snowplow dbt packages and tooling.
+To get a comprehensive overview of the out-of-the-box provided schemas, [please refer to this page](/docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md).
+
+Our [data product accelerators](https://snowplow.io/data-product-accelerators/) are a great resource to develop your data product faster using out-of-the-box data structures based on best practices.
+
+## Choose your naming convention
+
+Having a common standard for how to name events, entities and properties can make it easier to understand and extend the tracking design.
+The following are our recommendations, but in general, it's more important to be consistent across your schemas than to follow these recommendations:
+
+* Use snake case for both the schema and properties. This will ensure that the names are consistent with how the properties end up in the warehouse (for some warehouses, property names are converted to snake case regardless of how they are defined in the schemas). Avoid using hyphens to separate words, instead use underscores.
+* Use the verb – noun convention for event names. For instance, `add_to_cart`, `play_video`.
+* Be consistent about the tense (present or past). Make sure that all your events use the same tense (e.g., `play` and `pause` instead of `played` and `pause`).
+* Use singular in the entity name (e.g., `product` instead of `products`).
+
+## Start with entities
+
+A good practice when designing tracking is to start with [the entities](/docs/understanding-your-pipeline/entities/index.md).
+Entities contextualize and join events together, and they are often the levels of analysis a business is interested in (e.g., sale, user, organization, location).
+
+It may not be obvious whether some properties should be added to events or entities.
+In general, it is preferable to place information in entities as this will enable it to be reused across multiple events.
+It may often be the case that events do not have any properties – this is totally fine.
+Event properties should be limited only to information that is strictly related to the event and unlikely to be reused elsewhere (e.g., error message for an application error event).
```

Comment:
It might be worth calling out these _sort of_ map to entities being classic dimensions, and events map more to facts - not perfect but might help some people more classic data